website/snippets/_cloud-environments-info.md

```diff
@@ -34,6 +34,16 @@ Both development and deployment environments have a section called **General Set
 - If you select a current version with `(latest)` in the name, your environment will automatically install the latest stable version of the minor version selected.
 :::
 
+### Repo caching <Lifecycle status='Enterprise plan' /> 
```

Comment:
In other notes do we say "Plan"? Feel like this could be shortened to "Enterprise" unless we do this elsewhere

Reply:
just took a look in gh for prior art. there's actually several places where we use `enterprise plans`